statement.k

```diff
@@ -16,17 +17,77 @@ rule if (false) S:Statement => .
 rule if (true) S:Statement else S1:Statement => #exeStmt(S)
 rule if (false) S:Statement else S1:Statement => #exeStmt(S1)
 
-rule #exeStmt({ Ss:Statements }) =>  Ss 
+rule #exeStmt({ Ss:Statements }) =>  Ss
+
+rule #exeStmt(S:IfStatement) => S
+
+rule #exeStmt(S:WhileStatement) => S
 
-rule #exeStmt(S:IfStatement) => S 
 
-rule _:Value ; => . 
+rule _:Value ; => .
 
 rule
     <k> #exeStmt(S:SimpleStatement;) => S; ...</k>
     <callStack> .List </callStack>
 
 rule
-   #localFunctionCall(FN:Id, ARGS:Values) => functionCall(FN; ARGS)      [structural]
-   
+    #localFunctionCall(FN:Id, ARGS:Values) => functionCall(FN; ARGS)      [structural]
+
+rule [While]:
+    while (E:Expression) S:Statement => if (E) { S #loop_mark while (E) S }      [structural]
```

Comment:
As a style note, I personally prefer to wrap rules in the cell, eg:

```
rule [While]:
    <k> while (E) S => if (E) { S #loop_mark while (E) S } ... </k>
```

I think it makes the semantics more readable when they get large.

Reply:
Revised accordingly.